pkgs/development/compilers/llvm/10/default.nix

```diff
@@ -5,13 +5,25 @@
 }:
 
 let
-  release_version = "10.0.0";
-  version = release_version; # differentiating these (variables) is important for rc's
+  release_version = "10.0.1";
+  candidate = "rc1";
+  version = "${release_version}-${candidate}"; # differentiating these (variables) is important for RCs
   targetConfig = stdenv.targetPlatform.config;
 
-  fetch = name: sha256: fetchurl {
-    url = "https://github.com/llvm/llvm-project/releases/download/llvmorg-${release_version}/${name}-${version}.src.tar.xz";
```

Comment:
Might be interesting to know why the release tarballs were used in the first place.

Reply:
My wild guess would be, because originally the code was distributed over multiple repositories.

Reply:
This is now moot, as the splittar balls are back...